dags/reporting-dag.py

```diff
@@ -15,13 +15,12 @@
 AIRFLOW_PASSWORD = 'admin'
 AIRFLOW_WEBSERVER_HOST = 'webserver'
 AIRFLOW_WEBSERVER_PORT = '8080'
-GCP_CONN_ID= 'google_cloud_storage'
```

Comment:
Looks like the `headers` in ln 11 can be removed

Reply:
headers line removed